src/CMakeLists.txt

```diff
@@ -166,8 +166,22 @@ configure_file("${SM_XCODE_DIR}/plistHelper.in.hpp" "${SM_XCODE_DIR}/plistHelper
 if(APPLE)
   set(MACOSX_BUNDLE_BUNDLE_NAME ${SM_EXE_NAME})
   add_executable("${SM_EXE_NAME}" MACOSX_BUNDLE ${SMDATA_ALL_FILES_SRC} ${SMDATA_ALL_FILES_HPP})
-  set_target_properties("${SM_EXE_NAME}" PROPERTIES MACOSX_BUNDLE TRUE)
+  set_target_properties("${SM_EXE_NAME}" PROPERTIES
+    MACOSX_BUNDLE TRUE
+    XCODE_ATTRIBUTE_GCC_PREFIX_HEADER "${SM_SRC_DIR}/global.h"
```

Comment:
For those on Mac OS X (looking at you @dguzek ), does the prefix header show up for you as an arbitrary location like `src/archutils/Darwin/StepMania.pch`?